### PR TITLE
New xmleditor template

### PR DESCRIPTION
**Pull request by lfarrell**

### CODE REVIEW

File: static/js/xmleditor/templates/archives.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<mods:mods xmlns:mods="http://www.loc.gov/mods/v3"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <mods:titleInfo>
+    <mods:title></mods:title>
```
Comment (harringj) [inline static/js/xmleditor/templates/archives.xml]: Should there be a title here?

File: static/js/xmleditor/templates/archives.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  </mods:titleInfo>
+  <mods:identifier displayLabel="Accession Number" type="local"></mods:identifier>
+  <mods:identifier displayLabel="Digital Folder Number" type="local"></mods:identifier>
+  <mods:relatedItem type="original">
+    <mods:identifier displayLabel="Source Media Identifier"></mods:identifier>
+  </mods:relatedItem>
+  <mods:relatedItem type="isReferencedBy">
+    <mods:location>
+      <mods:url displayLabel="Link to finding aid"></mods:url>
```
Comment (harringj) [inline static/js/xmleditor/templates/archives.xml]: Should there be a url here?

File: static/js/xmleditor/templates/archives.xml (hunk 1 above)
Comment (lfarrell) [inline static/js/xmleditor/templates/archives.xml]: It's up to what UARMS wants. Template works fine.

File: static/js/xmleditor/templates/archives.xml (hunk 2 above)
Comment (lfarrell) [inline static/js/xmleditor/templates/archives.xml]: See above comment.

**Pull request merged**